Packs/ReversingLabs_Titanium_Cloud/Integrations/ReversingLabsTitaniumCloudv2/ReversingLabsTitaniumCloudv2.py

```diff
@@ -5,7 +5,7 @@
     RHA1Analytics, URIStatistics, URIIndex, AdvancedSearch, ExpressionSearch, FileDownload, FileUpload, \
     URLThreatIntelligence, AnalyzeURL, DynamicAnalysis, CertificateAnalytics
 
-VERSION = "v2.0.4"
+VERSION = "v2.0.5"
```

Comment:
Does the version change have no effect on the code?

Reply:
Same answer as above.